server/models/Profile.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+
+const ProfileSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+    },
+    gender: {
+      type: String,
+      enum: ['male', 'female'],
```

Comment:
```suggestion
      enum: ['male', 'female', 'non-binary'],
```
For individuals who don't identify as male/female 😄